notebooks/A_CNN_Annotation_with_Napari.ipynb

```diff
@@ -304,12 +310,19 @@
     "                patch_label = points_layer.properties['State'][idx]\n",
     "\n",
     "                # grab the image patch\n",
-    "                image_patch, flagged = get_image_patch(image_layers, patch_coords, shape=shape)\n",
+    "                image_patches = get_image_patch(image_layers, patch_coords, shape=shape)\n",
+    "                \n",
+    "                # check if the item was flagged:\n",
+    "                flagged = \"\"\n",
+    "                if image_patches[1] == True:\n",
+    "                    flagged += \"_flagged\"\n",
+    "                    export_data['flagged'][idx] = \"True\"\n",
```

Comment:
Can you not just use the boolean flag here? Or does that not work with the json writer?